agent/rpc/peering/service.go

```diff
@@ -830,7 +830,7 @@ func (s *Service) HandleStream(req HandleStreamRequest) error {
 
 			if resp := msg.GetResponse(); resp != nil {
 				// TODO(peering): Ensure there's a nonce
-				reply, err := s.processResponse(req.PeerName, req.Partition, resp)
```

Comment:
The PeeringList and PeeringReady endpoints in this file also need updating so that the count is actually emitted in the API responses.

Reply:
yes, that was going to come after getting this round of feedback (even potentially as a next PR)

Reply:
I added a reconciliation here: https://github.com/hashicorp/consul/pull/13718